settings.gradle.kts

```diff
@@ -15,11 +15,21 @@ pluginManagement {
 			includeGroup("net.fabricmc")
 			includeGroup("fabric-loom")
 		}
+		exclusiveMaven("https://maven.neoforged.net/releases") {
+			includeGroupByRegex("net\\.neoforged.*")
+			includeGroup("codechicken")
+			includeGroup("net.covers1624")
+		}
 		exclusiveMaven("https://repo.spongepowered.org/repository/maven-public/") {
 			includeGroupByRegex("org\\.spongepowered.*")
 		}
 		gradlePluginPortal()
 	}
+	plugins {
+		id("net.neoforged.gradle.common") version("7.0.80")
+		id("net.neoforged.gradle.userdev") version("7.0.80")
+		id("fabric-loom") version("1.1.+")
+	}
```

Comment:
The conflict between NeoForge and Forge may appear because these plugins are being applied at the root project level here. Can these be applied in the NeoForged gradle projects instead? Note there are no FG plugins set up in this file.